src/loader.rs

```diff
@@ -11,11 +16,243 @@ impl Loader {
     }
   }
 
-  pub(crate) fn load<'src>(&'src self, path: &Path) -> RunResult<&'src str> {
+  pub(crate) fn load_without_includes<'src>(&'src self, path: &Path) -> RunResult<&'src str> {
```

Comment:
I like just calling the base case `load`:

```suggestion
  pub(crate) fn load<'src>(&'src self, path: &Path) -> RunResult<&'src str> {
```